tests/test_token_tax.py

```diff
@@ -0,0 +1,102 @@
+"""Token tax module test examples on the BSC Chain for the ELEPHANT token
+
+To run tests in this module:
+
+.. code-block:: shell
+
+    export BNB_CHAIN_JSON_RPC="https://bsc-dataseed.binance.org/"
+    pytest -k test_token_tax
+
+"""
+import os
+
+import pytest
+from eth_typing import HexAddress, HexStr
+from web3 import  HTTPProvider, Web3
+from eth_defi.uniswap_v2.token_tax import estimate_token_taxes, LowLiquidityError
+
+from eth_defi.ganache import  fork_network
+from eth_defi.uniswap_v2.token_tax import TokenTaxInfo
+from eth_defi.uniswap_v2.deployment import UniswapV2Deployment, fetch_deployment
+
+
+SUSHISWAP_FACTORYV2 = "0xc35DADB65012eC5796536bD9864eD8773aBc74C4"
```

Comment:
These are still not oytest fixtures and commented as requested.

Please look other test modules for examples what's the best practice to have constants in pytest tests.

Reply:
Here is an example:

https://github.com/tradingstrategy-ai/web3-ethereum-defi/blob/master/tests/test_ganache.py#L31